pandas/core/arrays/datetimelike.py

```diff
@@ -65,6 +72,8 @@ def cmp_method(self, other):
 
 class AttributesMixin(object):
 
+    _scalar_types = (Period, Timestamp, Timedelta)
```

Comment:
shouldn't ``NaT`` also be here?

Reply:
It seems to be unused now. I'll remove it.